README.md

```diff
@@ -25,10 +25,19 @@ To address challenges 5 and 6, this repository contains a freely accessible, une
 
 ![A wallet architecture using Hierarchical Deterministic Keys associated with keys protected using a wallet secure cryptographic device, optionally using Asynchronous Remote Key Generation (ARKG).](media/deployment.svg)
 
+Please note that when ARKG is used, issuers have to store an additional secret key per user / holder.
+
```

Comment:
```suggestion
```

The issuer does not need to store secret keys. The issuer receives a public seed value, and uses it to create a key handle, which they communicate to the solution instance upon issuance. To keep the introduction lightweight I propose to keep this detail out.